Cargo.lock

```diff
@@ -866,11 +941,45 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "71655c45cb9845d3270c9d6df84ebe72b4dad3c2ba3f7023ad47c144e4e473a5"
 dependencies = [
  "bitflags 1.3.2",
- "clap_lex",
+ "clap_lex 0.2.4",
  "indexmap",
  "textwrap",
 ]
 
+[[package]]
+name = "clap"
```

Comment:
Where is this dep coming from?

Reply:
I don't know. I have added a cli app but it has no argument parsing at all.

Reply:
There is no longer a dependency on this.